doc/CONTRIBUTING.md

```diff
@@ -1,163 +0,0 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
People may be using this path actively in links and whatnot and us moving it will break.

I think we should just keep it as is.